### PR TITLE
Use {% do %} in template

### PR DESCRIPTION
**Pull request by beastbytes**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

**Comment by samdark** [comment]
Thanks. How it is better than the current variant?

File: src/Extensions/YiiTwigExtension.php [hunk 1]
```diff
@@ -31,6 +31,70 @@ public function getFunctions(): array
                 /** @return mixed */
                 fn (string $id) => $this->container->get($id),
             ),
+            new TwigFunction(
```
Comment (vjik) [inline src/Extensions/YiiTwigExtension.php:34]: Seems, we can split to several extensions (`ContainerExtension`, `ArrayHelperExtension`, ...)

Comment (rustamwin) [reply src/Extensions/YiiTwigExtension.php:34]: And I think it would be good to make these changes in separate PRs.

**Comment by rustamwin** [comment]
@beastbytes could you update from the master and leave only doc changes?

**Comment by rustamwin** [comment]
Closing in favor #52

**Pull request closed without merging**